lib/controllers/foods-controller.js

```diff
@@ -12,6 +13,22 @@ function show (request, response) {
     });
 }
 
+function create (request, response) {
+  var name = request.body.name
+  var calories = request.body.calories
+
+  var whatever = Food.insertFood(name, calories)
```

Comment:
Make sure you chain a `.then()` on your `insertFood()`. Since it's a promise, it won't actually execute unless you `.then` it. Typically, you'd want your response to be in the then, kind of like the commented out code below.